Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2095,6 +2095,12 @@ extension BrowserViewController: SettingsDelegate {
     }
 }
 
+extension BrowserViewController: EnhancedTrackingProtectionMenuDelegate {
+    func settingsOpenPage(settings: Route.SettingsSection) {
+        self.homePanelDidRequestToOpenSettings(at: .contentBlocker)
+    }
+}
+
```

Comment:
With the latest changes we've made, I think we can remove this protocol conformance on BrowserViewController